modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -91,3 +91,18 @@ Required parameters:
     </email:smtps-connection>
 </email:smtp-config>
 ----
+
+Additionally, if you experience SMTPS connection issues, check the following recommendations:
+
+If you have two-factor authentication set up:
+
+. There is no need for you to turn `Less Secure Apps` on
+. Generate an app specific password and use that one instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in using App Password] for details
```

Comment:
```suggestion
. Generate an app-specific password and use that instead of your normal password. 
See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details.
```